backend/services/llm/completion_service.py

```
@@ -30,6 +30,52 @@
+                        "description": tool_def.description,
+                        "parameters": tool_def.parameters.model_dump(exclude_none=True)
+                    }
+                }
+                
+                # Add mock_data if available for tool execution service
+                if tool_def.mock and tool_def.mock.enabled:
+                    tool_dict["mock_data"] = tool_def.mock.model_dump(exclude_none=True)
+                
+                tools.append(tool_dict)
+                
+            except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)